wear/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt

```diff
@@ -4,13 +4,16 @@ import android.app.Application
 import android.content.Intent
 import android.content.IntentFilter
 import dagger.hilt.android.HiltAndroidApp
+import io.homeassistant.companion.android.common.data.MTLSHelper
 import io.homeassistant.companion.android.sensors.SensorReceiver
 
 @HiltAndroidApp
 open class HomeAssistantApplication : Application() {
     override fun onCreate() {
         super.onCreate()
 
+        MTLSHelper().init(this)
+
```

Comment:
This call is there for two reasons:
* it uses "this" context to call getExternalFilesDir and obtain base path to load files
* it actually tries to import keys if exists
If we use hilt (which i was not aware of) we could postpone import on first usage on WebClient or first OkHttp call.
How am i supposed to obtain base dir there? I'm not an Android expert, as you can see.

Reply:
> * it uses "this" context to call getExternalFilesDir and obtain base path to load files

You can pass in the context via hilt


> * it actually tries to import keys if exists
>   If we use hilt (which i was not aware of) we could postpone import on first usage on WebClient or first OkHttp call.
>   How am i supposed to obtain base dir there? I'm not an Android expert, as you can see

This is why I was saying we should probably move to a file picker.  It could prompt for the keys after the app tries to authenticate and sees the need for them.